packages/core/helper-plugin/lib/src/components/RelationsInput/components/ShadowList.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+
+const ShadowBox = styled(Box)`
```

Comment:
I'd still recommend `RelationList` as name, because this is what it does: it holds a list of relations. The shadows are visual side-effects and shouldn't therefore be part of the name imo.

Wdyt?

Reply:
sounds good!